ratis-server/src/main/java/org/apache/ratis/server/impl/SnapshotInstallationHandler.java

```diff
@@ -353,4 +354,4 @@ private RoleInfoProto getRoleInfoProto(RaftPeer leader) {
         .setFollowerInfo(followerInfo)
         .build();
   }
-}
\ No newline at end of file
+}
```

Comment:
@hemantk-12 , could you revert this whitespace change?  I tried to do in on the GitHub web ui but it did not work.

Reply:
Removed.